roles/authenticator_maps/meta/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+galaxy_info:
+  role_name: "authenticator_maps"
+  author: "Martin Slemr"
+  description: "An Ansible Role to create authenticator maps in automation platform gateway."
+  company: "Red Hat"
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+  license: "GPLv3"
+
+  min_ansible_version: "2.14"
```

Comment:
Need to be consistent with min Ansible version, also I think the technically correct way to pick a version is without quotes and be 3 levels like 2.16.0